sign_sysext_keyvault.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -euo pipefail
```
We should stop script execution on errors